lib/logger.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// var exists = require('101/exists');
+// var isObject = require('101/is-object');
+var bunyan = require('bunyan');
+// var Bunyan2Loggly = require('bunyan-loggly').Bunyan2Loggly;
+
+/**
+ * Bunyan logger for ponos.
+ * @author Bryan Kendall
+ * @module ponos:logger
+ */
+module.exports = bunyan.createLogger({
```

Comment:
Should the logger should be a `Server` instance variable? If we did it that way we could provide accessor and mutators to allow for easy mutation and full swap-out depending on what makes sense in the given implementation.

On the flip side it would mean we'd have to pass it from the server to each of the workers, and the workers to errors... uugghh.